app/jobs/whitehall_document_import_job.rb

```diff
@@ -15,10 +15,19 @@ class WhitehallDocumentImportJob < ApplicationJob
   def perform(document_import)
     document_import = WhitehallImporter::Import.call(document_import)
     document_import = WhitehallImporter::Sync.call(document_import)
+
+    if document_import.completed?
+      GdsApi.whitehall_export.document_migrated(document_import.whitehall_document_id)
+    end
+
     document_import.whitehall_migration&.check_migration_finished
```

Comment:
Did we plan to include this as part of error handling too or am I thinking of a different story or PR?

Reply:
Different PR 👍